acceptance/tests/windows/prevent_password_changes.rb

```diff
@@ -0,0 +1,76 @@
+require 'date'
+require 'puppet/acceptance/windows_utils'
+extend Puppet::Acceptance::WindowsUtils
+
+def random_username
+  "pl#{rand(999999).to_i}"
+end
+
+def change_password_manifest(username)
+  return <<-MANIFEST
+    user { '#{username}':
+      ensure   => 'present',
+      password => 'Password-#{rand(999999).to_i}'
+    }
+  MANIFEST
+end
+
+OLD_PASSWORD='0ldP@ssword'
+
+test_name 'PUP-6569 Puppet should not reset passwords for disabled, expired, or locked out Windows user accounts' do
```

Comment:
Nit: `test_name`'s usually the first thing defined in the file -- defs and requires come after. See e.g. https://github.com/puppetlabs/puppet/blob/master/acceptance/tests/resource/service/AIX_service_provider.rb or https://github.com/puppetlabs/puppet/blob/master/acceptance/tests/resource/service/smf_basic_tests.rb